cluster/gce/gci/configure-helper.sh

```diff
@@ -2723,7 +2737,7 @@ EOF
   if [[ "${ENABLE_NODE_LOGGING:-}" == "true" ]] && \
      [[ "${LOGGING_DESTINATION:-}" == "elasticsearch" ]] && \
      [[ "${ENABLE_CLUSTER_LOGGING:-}" == "true" ]]; then
-    setup-addon-manifests "addons" "fluentd-elasticsearch"
+    setup-addon-manifests-remote "addons" "fluentd-gcp" "https://github.com/kubernetes-sigs/instrumentation-addons/tree/master/fluentd-elasticsearch"
```

Comment:
curious, why the name change `fluentd-elasticsearch` -> `fluentd-gcp`? This PR github.com/kubernetes-sigs/instrumentation-addons/pull/1 only added `fluentd-elasticsearch` directory

Reply:
sorry, copy paste fuckup. i'll fix it...

Reply:
Should we really be installing from HEAD of the master branch like this?
This is also a headache waiting to happen when the repo does http://k8s.dev/rename

Reply:
agreed. I don't think we should switch addons to pulling from remote sources like this.

Reply:
@liggitt there has been a long standing ask to move everything out of `k/k/cluster`.  This PR and one other is moving addons from here to `k8s-sigs/sig-instrumentation-addons` repo.  If we do not want to move the addons out individually, is there a plan for how to move addons out as a whole?  

ie, should there be an `addon` repository in the kubernetes-sigs org?  Or is this a topic that needs a broader discussion somewhere?

Reply:
leaving `/cluster` in place but making it load content remotely doesn't satisfy the "remove cluster" request, and makes `/cluster` as used by e2e tests across branches less stable.

Reply:
my thought was that `/cluster` would get removed piecemeal like this.  so this would be part one of `n`.  

would using the suggestion of not installing from HEAD help with the stability concern?  

if thats still a no-go, is there a WG or something for how to move `/cluster` to someplace else?

Reply:
@liggitt is there a path forward for this?  If not, I'm going to close this PR and another one and unwind some other work that got done.

and what would be the right next venue for how to move `addons` as a whole out of k/k, if you know?

Reply:
@coffeepac can we cut release tags in that repo and then bump as needed here?

Reply:
we certainly can but I'm not sure that liggit was okay with having individual addons be out-of-tree from the addon-manager.  

this seems to need a larger discussion around what is and isn't acceptable for items in cluster as they migrate out.

Reply:
I guess I would expect straight removal of the addon from `cluster`, rather than adding remote-install capabilities to `cluster`. If there are tests exercising the addon, I would expect those tests to install the version they want to test

Reply:
so, essentially, move the remote-install code to tests that exercise the addon and have it install a pinned version.

that's fair and shouldn't be too difficult to do.  I'll take a look in the next few days at this.

Reply:
please ping when done for review. Also this still uses name `fluentd-gcp`